matter/wifi/wfx_host_events.h

```diff
@@ -156,11 +154,15 @@ typedef enum {
 } sl_wfx_interface_t;
 
 #endif /* RS911X_WIFI */
+
 #ifdef __cplusplus
 extern "C" {
 #endif
 
 void sl_wfx_host_gpio_init(void);
+void sl_wfx_host_init_bus(void);
+void sl_wfx_host_reset_chip(void);
```

Comment:
Where were those function declared before ?

Reply:
`/third_party/silabs/gecko_sdk/platform/radio/wifi/wfx_fmac_driver/sl_wfx_host_api.h`

Is where those are defined. They changed the prototype from sl_status_t to void so it doesn't link anymore I guess. Why is this done ?

Reply:
This seems really bad. either use the right one from the gsdk or rename then with something else. I feel there is WAY too many implementation of functions with the same name..